stable/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -61,6 +61,9 @@ spec:
       {{- end }}
       securityContext:
         runAsUser: 0
+      {{- if .Values.rbac.install }}
+      serviceAccountName: {{ template "fullname" . }}
+      {{- end }}
```

Comment:
Let's do `serviceAccountName: {{ if .Values.rbac.install }}{{ template "fullname" . }}{{ else }}"{{ .Values.rbac.serviceAccountName }}"{{ end }}`. Then add `rbac.serviceAccount=default` to `values.yaml`. This supports Helm creating the ServiceAccount & RBAC, a user externally creating a ServiceAccount & RBAC, or simply using the `default` ServiceAccount.

It's a pattern introduced in #1504 and being followed in other PRs.